src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types/types';
+
+interface Props {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
user should be required, it doesn't make sense to render this component without user